setup.py

```diff
@@ -24,12 +24,13 @@
     maintainer="SUSE QA Maintenance",
     maintainer_email="qa-maintenance@suse.de",
     license="License :: Other/Proprietary License",
-    platforms=["Linux", "Mac OSX"],
```

Comment:
Why remove this?

Reply:
Because it really never worked on MacOS 😃 
Dependency on rpm-python can't be really solved as on pypi is only placeholder empty package and real bindings depend on rpm distributed in Linux distributions and none of providers on mac doesn't deliver it